aform/src/components/form/AFieldset.vue

```diff
@@ -4,7 +4,9 @@
 			{{ label }}
 			<CollapseButton v-if="collapsible" :collapsed="collapsed" />
 		</legend>
-		<AForm v-show="!collapsed" :schema="schema" :data="formData" />
+		<slot>
+			<AForm v-show="!collapsed" :schema="schema" :data="formData" />
+		</slot>
```

Comment:
I think the `slot` should also expose the `collapsed` prop, since that's not available for the user

```suggestion
		<slot :collapsed="collapsed">
			<AForm v-show="!collapsed" :schema="schema" :data="formData" />
		</slot>
```

Reply:
I tried that and it doesn't work. I need to find the docs that say this, but it's not allowed. I couldn't come up with an alternative that I liked, maybe it needs a wrapper?

Reply:
This was the docs I was referring to: https://vuejs.org/guide/components/slots.html#scoped-slots. Not sure why it wouldn't work though.

Reply:
This looks like its worth a try.